apps/design-land/src/app/sidebar/sidebar.module.ts

```diff
@@ -3,7 +3,7 @@ import { CommonModule } from '@angular/common';
 
 import { SidebarRoutingModule } from './sidebar-routing.module';
 import { SidebarComponent } from './sidebar.component';
-import { DaffSidebarModule, DaffButtonModule, DaffNativeSelectModule, DaffFormFieldModule } from '@daffodil/design';
+import { DaffSidebarModule, DaffButtonModule, DaffNativeSelectModule, DaffFormFieldModule, DaffLinkSetModule } from '@daffodil/design';
```

Comment:
DaffLinkSetModule is added but removed from the example usage?

Reply:
fixed in ca31103737ff13c2b0e15942fc3032ead77086da